jenkins_jobs/update_versions_readme/script.sh

```diff
@@ -0,0 +1,9 @@
+python host-os.py \
+  --verbose \
+  update-versions-readme \
+    --build-versions-repository-url ${VERSIONS_REPO_URL} \
+    --build-version ${VERSIONS_REPO_COMMIT} \
+    --updater-name "${GITHUB_BOT_NAME}" \
+    --updater-email "${GITHUB_BOT_EMAIL}" \
+    --push-repo-url "ssh://git@github.com/${GITHUB_BOT_USER_NAME}/versions.git" \
```

Comment:
why you hardcode git@github.com and versions.git here and not in build-versions-repository-url value? Please be consistent